### ISSUE TITLE
Bug: Loading Modules Issue...

### ISSUE THREAD

**Issue by Nefariis** [Lady-Binary/EasyLoU#161]
I think something changed after version 1.2.1.0, I can no longer load modules the way that i used too.

Simple Repro 

-- in module.lua --
```
local test = {}

function test.world()
    print("hello world")
end

return test
```

-- in main.lua --
```
local test = require("module")
test.word()
```

This works as expected with the basic Lua interpreter but it is failing in MoonSharp with the error message - `libfunc_require:(12,1-44): module 'module' not found`. 

What's more is it looks like the variables LUA_PATH and package.path are nil in MoonSharp and even when you try and set the package.path manually a la `package.path = "./?.lua;../?.lua"` - I still receiving the error message. 

If I had to guess it has to do with this - https://www.moonsharp.org/scriptloaders.html and specifically this -

```
((ScriptLoaderBase)script.Options.ScriptLoader).ModulePaths = new string[] { "MyPath/?", "MyPath/?.lua" };
// or
((ScriptLoaderBase)script.Options.ScriptLoader).ModulePaths = ScriptLoaderBase.UnpackStringPaths("MyPath/?;MyPath/?.lua");
```
but I honestly dont know enough anything about  MoonSharp - would you mind taking a look?

**Comment by Nefariis** [comment]
ahhh just noticed i was on 1.2.1.0 and it was fixed in 1.2.1.1 - I just upgraded to the latest and now everything works 🤦‍♂️

**Comment by Lady-Binary** [comment]
thanks for the update!
